### PR TITLE
Initial config

### PR DESCRIPTION
**Pull request by mangelats**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```
Comment (manelclos) [inline LICENSE]: 2018 Microdisseny Giscube SLU

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+-----------
+
+1. Add "data_access_log" to your INSTALLED_APPS setting like this::
+
+    INSTALLED_APPS = [
+        ...
+        'data_access_log',
+    ]
+
+2. Run `python manage.py migrate` to create the polls models.
```
Comment (manelclos) [inline README.rst]: polls models?

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+3. Add the `@logged` decorator to any viewset you'd like to log::
+
+    from data_access_log.decorators import logged
+
+    @logged
```
Comment (manelclos) [inline README.rst]: what about @log_access ?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+setup(
+    name='django_data_access_log',
+    version=VERSION,
+    packages=find_packages(),
+    include_package_data=True,
+    license='MIT License',
+    description='Log the acess to some endpoints of your API',
+    long_description=README,
+    url='http://www.microdisseny.com/',
+    author='Microdisseny SLU',
```
Comment (manelclos) [inline setup.py]: Microdisseny Giscube SLU

**Review by manelclos — COMMENTED** [review]
please review comments

File: README.rst (hunk 3 above)
Comment (mangelats) [reply README.rst]: Decorator should be a property of the function they modify (as far as i understand that's the usual way of doing it). Maybe `@logs_access`

Comment (manelclos) [reply README.rst]: @log_data_access ?

**Review by abusquets — APPROVED** [review]

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+1. Add "data_access_log" to your INSTALLED_APPS setting like this::
+
+    INSTALLED_APPS = [
+        ...
+        'data_access_log',
+    ]
+
+2. Run `python manage.py migrate` to create the models.
+
+3. Add the `@` decorator to any viewset you'd like to log::
```
Comment (manelclos) [inline README.rst]: 3. Add the `@data_access_log` decorator to any viewset you'd like to log:

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,24 @@
+=====
+Django data access log
+=====
+
+Django data access log is a simple Django app to log the access into a JRF viewset.
```
Comment (manelclos) [inline README.rst]: Django data access log is a simple Django app to log the data accessed in a DRF viewset.

File: README.rst [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+    INSTALLED_APPS = [
+        ...
+        'data_access_log',
+    ]
+
+2. Run `python manage.py migrate` to create the models.
+
+3. Add the `@` decorator to any viewset you'd like to log::
+    class MyLoggedViewSet(ModelViewSet):
```
Comment (manelclos) [inline README.rst:24]: include @data_access_log in the example code

File: setup.py [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='django_data_access_log',
+    version=VERSION,
+    packages=find_packages(),
+    include_package_data=True,
+    license='MIT License',
+    description='Log the acess to some endpoints of your API',
```
Comment (manelclos) [inline setup.py]: description='Log the access to endpoints of your API',

File: setup.py [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    long_description=README,
+    url='http://www.microdisseny.com/',
```
Comment (manelclos) [inline setup.py]: url='https://github.com/Microdisseny/django-data-access-log/',

File: setup.py [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+    author='Microdisseny Giscube SLU',
+    author_email='info@microdisseny.com',
```
Comment (manelclos) [inline setup.py]: author_email='tech@microdisseny.com',

**Review by manelclos — CHANGES_REQUESTED** [review]
Please adjust the last minute fixes

File: README.rst (hunk 7 above)
Comment (mangelats) [reply README.rst:24]: This is modified in the following commit

Comment (manelclos) [reply README.rst:24]: ok, maybe next time include the full/right version here.

**Review by manelclos — APPROVED** [review]

**Pull request merged**